developer-tools/scaleway-sdk/go-sdk.mdx

```diff
@@ -0,0 +1,218 @@
+---
+meta:
+  title: Scaleway Go software developement kit (SDK)
+  description: This page explains how to install and configure the Scaleway CLI locally
+content:
+  h1: Scaleway Go software developement kit (SDK)
+  paragraph: This page explains how to install and configure the Scaleway CLI locally
+tags: devtools cli download-cli scaleway-cli command-line 
+dates:
+  validation: 2023-09-21
+  posted: 2021-09-21
+categories:
+  - devtools
+---
+
+The Scaleway Go SDK is a tool provided by Scaleway for developers to interact with its services programmatically. 
+It allows for the automation of tasks like server management, storage operations, and networking setup through Go scripts, handling the complexities of direct API calls and authentication behind simple Go function calls. 
+This makes it easier to integrate Scaleway's cloud resources into applications or manage them through code. 
+
+
+## Installation of the Go SDK
+
+You can install the SDK by using the following command: 
+
+```
+go get github.com/scaleway/scaleway-sdk-go
+```
+
+<Message type="note">
+  This library is made to work with Go 3.8 but should work on the latest versions too.
+</Message>
+
+## Initialization
+
+You need a pair of access and secret keys to connect to Scaleway API. Check out the documentation to find out [how to retrieve them](/identity-and-access-management/iam/how-to/create-api-keys/).
+
+`scaleway` APIs must be initialized with a `scaleway.Client`.
+
+A minimal setup would look like this:
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/scaleway-sdk-go/utils"
```

Comment:
cant find the utils module when running the file. Is this up-to-date? cc @remyleone 🙏

Reply:
Also, it works fine without "github.com/scaleway/scaleway-sdk-go/utils"

Reply:
```suggestion
```